### PR TITLE
Homework3

### PR DESCRIPTION
**Pull request by akuzmichova**
Hi Deepak, can you please review

### CODE REVIEW

File: src/test/java/Class3/Homework3.java [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+        String lowTempValueWithDegree = lowTempValue.getText();
+        String lowTempValueNoDegree = lowTempValueWithDegree.substring(0, lowTempValueWithDegree.length()-1);
+        int lowTemp = Integer.parseInt(lowTempValueNoDegree);
+
+        WebElement highTempValue = driver.findElement(By.className("high-temp-text"));
+        String highTempValueWithDegree = highTempValue.getText();
+        String highTempValueNoDegree = highTempValueWithDegree.substring(0, highTempValueWithDegree.length()-1);
+        int highTemp = Integer.parseInt(highTempValueNoDegree);
+
+        if (flTemp >= lowTemp && flTemp <= highTemp) {
```
Comment (deepaktechnosoft) [inline src/test/java/Class3/Homework3.java:33]: Instead if-else use Assert to verify the testcase

File: src/test/java/Class3/Homework3.java [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+        driver.get("https://facebook.com/");
+
+        driver.findElement(By.linkText("Create New Account")).click();
+        try {
+            Thread.sleep(1000);
+        } catch (InterruptedException e) {
+            e.printStackTrace();
+        }
+        driver.findElement(By.name("firstname")).sendKeys("First");
+        try {
```
Comment (deepaktechnosoft) [inline src/test/java/Class3/Homework3.java:84]: why sleep here??

Comment (akuzmichova) [reply src/test/java/Class3/Homework3.java:84]: Test was failing without sleep, i believe it is my internet connection speed

**Review by deepaktechnosoft — APPROVED** [review]

File: src/test/java/Class3/Homework3.java (hunk 1 above)
Comment (deepaktechnosoft) [reply src/test/java/Class3/Homework3.java:33]: boolean result = flTemp >= lowTemp && flTemp <= highTemp;
Assert.assertTrue(  result , "Feels like temp is less or greater than low-high temp"   ) ;

**Pull request merged**